src/iomusxml.cpp

```diff
@@ -1323,6 +1325,44 @@ void MusicXmlInput::ReadMusicXmlNote(pugi::xml_node node, Measure *measure, int
         }
     }
 
+    // mordent
+    pugi::xpath_node xmlMordent = notations.node().select_single_node("ornaments/mordent");
+    if (xmlMordent) {
+        Mordent *mordent = new Mordent();
+        m_controlElements.push_back(std::make_pair(measureNum, mordent));
+        mordent->SetStaff(staff->Att::StrToXsdPositiveIntegerList(std::to_string(staff->GetN())));
+        mordent->SetStartid("#" + element->GetUuid());
+        // color
+        std::string colorStr = GetAttributeValue(xmlMordent.node(), "color");
+        if (!colorStr.empty()) mordent->SetColor(colorStr.c_str());
+        // form
+        mordent->SetForm(mordentLog_FORM_norm);
+        // long
+        std::string elongation = GetAttributeValue(xmlMordent.node(), "long");
+        if (!elongation.empty()) mordent->SetLong(ConvertWordToBool(elongation.c_str()));
+        // place
+        std::string placeStr = GetAttributeValue(xmlMordent.node(), "placement");
+        if (!placeStr.empty()) mordent->SetPlace(mordent->AttPlacement::StrToStaffrel(placeStr.c_str()));
+    }
+    pugi::xpath_node xmlMordentInv = notations.node().select_single_node("ornaments/inverted-mordent");
+    if (xmlMordentInv) {
+        Mordent *mordent = new Mordent();
+        m_controlElements.push_back(std::make_pair(measureNum, mordent));
+        mordent->SetStaff(staff->Att::StrToXsdPositiveIntegerList(std::to_string(staff->GetN())));
+        mordent->SetStartid("#" + element->GetUuid());
+        // color
+        std::string colorStr = GetAttributeValue(xmlMordentInv.node(), "color");
+        if (!colorStr.empty()) mordent->SetColor(colorStr.c_str());
+        // form
+        mordent->SetForm(mordentLog_FORM_inv);
```

Comment:
Actually one comment: do you really want this by default? Couldn't it remain unset (NONE)?